browser/src/control/Control.UIManager.js

```diff
@@ -457,6 +458,9 @@ L.Control.UIManager = L.Control.extend({
 
 	refreshNotebookbar: function() {
 			var selectedTab = $('.ui-tab.notebookbar[aria-selected="true"]').attr('id') || 'Home-tab-label';
+
+			this.refreshingNotebookbar = true;
+			var wasNotebookbarLoadedOnCore = this.notebookbar && this.notebookbar._isNotebookbarLoadedOnCore;
```

Comment:
there is method for that: `notebookbar.isInitializedInCore()` to not expose internals outside